spec/models/piece_spec.rb

```diff
@@ -93,5 +93,69 @@
       expect(pawn1.obstructed_horizontal?([4, 1])).to eq false
     end
 
-  end
+  end # end describe obstructed_horizontal
+
+  describe "#obstructed_vertical" do
+
+    before(:example) do
+      @game = create(:game)
+    end
+
+    it "should return true if piece is obstructed above" do
+      pawn1 = @game.pieces.create({
+        color: "white",
+        x_pos: 1,
+        y_pos: 1,
+        captured: false
+      })
+      pawn2 = @game.pieces.create({
```

Comment:
see how `let` can help you `DRY` some of these specs: https://relishapp.com/rspec/rspec-core/v/3-3/docs/helper-methods/let-and-let